### PR TITLE
Added send to transfer list option by generalising details panel button press

### LINKED ISSUE #4: Add a "send to transfer list" shortcut
Requested by /u/No_Bots.

### PR DESCRIPTION
**Pull request by Huskie**
Made a generic panel button function, controlled by param. Helped with the send to transfer list, keeps it DRY fixes #4

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -152,27 +155,32 @@
...
 
     /**
-     * Stores the current item in your club.
+     * Presses a button in the item details panel, based on a given button label.
      */
-    function storeInClub() {
+    function pressDetailsPanelButton(buttonLabel) {
         if (navigator.language.indexOf('en') !== 0) {
-            alert('The "Send to My Club" shortcut is only available when the app is in English. Blame EA!');
+            alert('The ' + buttonLabel + ' shortcut is only available when the app is in English. Blame EA!');
```
Comment (martellaj) [inline script.js]: nit: Can you please put quotes around the label?

File: script.js [hunk 2]
```diff
@@ -152,27 +155,32 @@
...
+            return;
+        }
+
+        if (typeof buttonLabel !== 'string') {
+            log('pressDetailsButton function failed: invalid button label parameter');
             return;
         }
 
-        log('Attempting to store current item in the club...');
+        log('Attempting to ' + buttonLabel);
```
Comment (martellaj) [inline script.js]: nit: Can you update this message to be: 
```javascript
`Attempting to press ${buttonLabel} button...`
```

File: script.js [hunk 3]
```diff
@@ -152,27 +155,32 @@
...
-            // Tap "Send to My Club" button.
+            // Tap the relevant button.
             const buttonArray = getDetailsPanelButtons();
-            const sendToMyClubButton = buttonArray.filter((button) => button.innerText.indexOf('Send to My Club') > -1)[0];
-            tapElement(sendToMyClubButton);
+            const button = buttonArray.filter((button) => button.innerText.indexOf(buttonLabel) > -1)[0];
+            tapElement(button);
         } catch (error) {
-            log('Unable to locate "Send to My Club" button.', true /* isError */);
+            log('Unable to locate ' + buttonLabel + ' button', true /* isError */);
```
Comment (martellaj) [inline script.js]: nit: Can you add in the quotes around the button label?

File: script.js [hunk 4]
```diff
@@ -152,27 +155,32 @@
...
             return;
         }
 
-        log('Successfully stored current item in the club.');
+        log('Successfully ' + buttonLabel);
```
Comment (martellaj) [inline script.js]:
```javascript
log(`Successfully pressed ${buttonLabel} button.`).
```

**Comment by Huskie** [comment]
@martellaj no problem, I am the very same. I've made those changes now 👍

**Comment by martellaj** [comment]
Thanks, @Huskie! Merged.

**Pull request merged**